### DISCUSSION TITLE
horizontal regridding with regrid2 yields NAN values

### DISCUSSION THREAD

**Discussion by ShihengDuan** [xCDAT/xcdat#598] [2. Questions]
### Question criteria

- [X] I added a descriptive title here.
- [X] I searched the [xCDAT GitHub Discussions](https://github.com/xCDAT/xcdat/discussions) to find a similar question and didn't find it.
- [X] I searched the [xCDAT documentation](https://xcdat.readthedocs.io/en/stable/) and/or [Xarray documentation](https://docs.xarray.dev/en/stable/).

### Describe your question

I was trying to regrid from a high resolution to coarse resolution with regrid2 but get lots of NAN values. It seems the documentation also shows some missing values here: https://xcdat.readthedocs.io/en/latest/examples/regridding-horizontal.html (section6)

code:
```
regrid2_gridder = Regrid2Regridder(ua_grid, vic_grid)
xesmf_gridder = XESMFRegridder(ua_grid, vic_grid, method='conservative')
year=2005
ua_data = xc.open_dataset('ori_data/4km_SWE_Depth_WY'+str(year)+'_v01.nc')
ua_data = ua_data.sel(time=ua_data.time.dt.month==3)
ua_data.lat.attrs['standard_name'] = 'Y'
ua_data.lon.attrs['standard_name'] = 'X'
regrid2_swe = regrid2_gridder.horizontal("SWE", ua_data)
regrid2_swe.SWE.isel(time=10).plot()
xesmf_swe = xesmf_gridder.horizontal("SWE", ua_data)
xesmf_swe.SWE.isel(time=10).plot()
```

Output:
<img width="1143" alt="image" src="https://github.com/xCDAT/xcdat/assets/26401994/40f43934-91be-4c3a-9ba4-de905f7c2229">
<img width="853" alt="image" src="https://github.com/xCDAT/xcdat/assets/26401994/53b69647-a572-40cd-8b56-1838a0cf74e2">

**Comment by pochedls** [comment]
@ShihengDuan – this seems like an issue (that is distinct from #580). I think you can change this to an issue (instead of a discussion). Could you create a minimal, complete, reproducible example?

Also – is this dataset rectilinear (e.g., SWE[lat, lon])?

**Reply by ShihengDuan** [reply]
Yes it is a lat-on dataset. I'll upload a slice of data and submit an issue soon.
